imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -12,57 +12,143 @@ import { setDrawerState } from '../general-actions'
 import Units, { collectionName } from '../../api/units'
 import UnitMetaData from '../../api/unit-meta-data'
 import FloatingActionButton from 'material-ui/FloatingActionButton'
+import { Tabs, Tab } from 'material-ui/Tabs'
+import SwipeableViews from 'react-swipeable-views'
+
+function ActiveDisabledFilteredUnits (props) {
+  const { filteredUnits, dispatch } = props
+  const currentUser = Meteor.userId()
+  const unitsByRoles = filteredUnits.filter(x => x.metaData.ownerIds)
+  const administratorUnits = unitsByRoles.filter(x => x.metaData.ownerIds.includes(currentUser))
+  const participantUnits = unitsByRoles.filter(x => !x.metaData.ownerIds.includes(currentUser))
+  const unassignedUnits = filteredUnits.filter(a => administratorUnits.filter(b => a.id === b.id).length === 0)
+  const unitsInvolvedIn = unassignedUnits.concat(participantUnits)
```

Comment:
```
const administratedUnits = filteredUnits.filter(unit => unit.metaData.ownerIds && unit.metaData.ownerIds.includes(currentUser))
const unitsInvolvedIn = filteredUnits.filter(unit => !unit.metaData.ownerIds || !unit.metaData.ownerIds.includes(currentUser))
```
**or**
```
const unitLists = filteredUnits.reduce((lists, unit) => {
  if (unit.metaData.ownerIds && unit.metaData.ownerIds.includes(currentUser)) {
    lists.administrated.push(unit)
  } else {
    lists.involvedIn.push(unit)
  }
  return lists
}, {administrated: [], involvedIn: []})
```
?

Reply:
> could not filter through the unit list that included units without ownerId metadata

Just for the record I can't think of any scenario in the production environment where a unit has no `ownerId`, this is due to the way units are created in Unee-T.
I understand that we might want to plan for this but this is a very, very, very unlikely case (to have units with no information about `ownerId`

Reply:
@franck-boullier In the production environment it might not be likely, but we still need to keep our testing envs running smoothly. Regardless, either one of the snippets I provided above should work on all envs.

Reply:
If we already have a solution then not a problem.
If this is a show stopper, or if having units with no ownerId information is a problem, then I'd suggest that we add an `ownerId` information to all the records in the DEV environment to make sure that the DEV envo reflects the reality of how things should be

Reply:
That wouldn't be enough, as PR apps are using DEV as their BZ server, but with a separate MongoDB instance. @sophieyang0813 's local env works in a similar way. However, we have solutions, and this is not a show stopper :smile: